src/Client/KvkPaginator.php

```diff
@@ -7,7 +7,7 @@
 use Werkspot\KvkApi\Client\Exception\PageDoesNotExistException;
 use Werkspot\KvkApi\Client\Profile\Company as ProfileCompany;
 
-final class KvkPaginator implements KvkPaginatorInterface
+class KvkPaginator implements KvkPaginatorInterface
```

Comment:
The client developer can create a mock using the interface, so this class doesn't need to be not-final, for such a reason.

Reply:
You're right this class can be final and we can mock it using the interface. I'll revert this change